db/schema.rb

```diff
@@ -22,8 +21,8 @@
     t.string "involvement_type"
     t.boolean "has_partner_opponent"
     t.boolean "receives_qualifying_benefit"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at", precision: nil, null: false
```

Comment:
@stephenrichards these schema changes look similar to the ones from my rubocop PR